R/check.R

```diff
@@ -23,3 +23,30 @@ check_pkgdown <- function(pkg = ".") {
     "v" = "No problems found in {pkgdown_config_href({pkg$src_path})}"
   ))
 }
+
+check_built_site <- function(pkg = ".") {
+  pkg <- as_pkgdown(pkg)
+
+  cli::cli_rule("Checking for problems")
+  index_path <- path_index(pkg)
+  if (!is.null(index_path)) {
+    check_missing_images(pkg, index_path, "index.html")
+  }
+}
+
+check_missing_images <- function(pkg, src_path, dst_path) {
```

Comment:
why not check all relative links? thinking of #2216

Reply:
this could be helpful for people trying to add files via assets.

Reply:
maybe using `get_site_paths()` (although that would prevent users from linking to stuff in the dev website, now probably linking to stuff in the dev website isn't a good idea?)

Reply:
Yeah, I think we'll probably accumulate more checks in here, now that we have this function.